nixos/modules/config/shells-environment.nix

```diff
@@ -107,14 +107,28 @@ in
     };
 
     environment.shellAliases = mkOption {
-      default = {};
-      example = { ll = "ls -l"; };
+      default = {
+        ls = "ls --color=tty";
+        ll = "ls -l";
+        l  = "ls -alh";
+      };
+      example = { gst = "git status"; };
       description = ''
         An attribute set that maps aliases (the top level attribute names in
         this option) to command strings or directly to build outputs. The
         aliases are added to all users' shells.
+
+        <note><para>
```

Comment:
@danbst
Yes I have build the manual, and everything looks good.
However, I couldn't find any documentation on the available tags.
I used existing options as a reference.